Makefile

```diff
@@ -4,7 +4,10 @@ include .env
 
 default: up
 
-DRUPAL_ROOT ?= /var/www/html/web
+APP_ROOT ?= /var/www/html
+DRUPAL_ROOT ?= $(APP_ROOT)/docroot
+DRUPAL_SITE_DIR ?= $(APP_ROOT)/sites/default
```

Comment:
We should kill this line - it's not doing anything at the mo and it's incorrect

Reply:
Hey! This should be fixed now to point to the correct place as w discussed :)

Reply:
Let's get this into the master and then we can migrate master into the bitbucket private repo (merge with site repo)